lib/config/tests.js

```diff
@@ -4,6 +4,11 @@ var defaultConfig = require('./defaults');
 module.exports = _.defaultsDeep({
   defaults: {
     vhosts: {
+      connection: {
+        options: {
+          heartbeat: 50,
+        },
+      },
       namespace: true,
```

Comment:
Not sure why this caused tests to fail. Rascal defaults the heartbeat to 10 which is quite a long time in seconds. However it could be that some of the load or flow control tests are filling the event loop, meaning that the broker doesn't get a response. If that's the problem then this was a really good find. Wouldn't even has crossed my mind.

Reply:
Yep! That could be the most obvious reason. It was failing randomly at various tests.. sometimes in Publications sometimes in Subscriptions.